pkg/pod/workingdir_init_test.go

```diff
@@ -28,6 +28,11 @@ import (
 const shellImage = "shell-image"
 
 func TestWorkingDirInit(t *testing.T) {
+	volumeMounts := []corev1.VolumeMount{{
```

Comment:
`implicitVolumeMounts` also includes the home mount (`/builder/home`)

Otherwise LGTM!

Reply:
Thanks! I have added the `home` in there,